src/test/java/com/ookawara/book/application/mapper/BookMapperTest.java

```diff
@@ -150,4 +151,21 @@ class BookMapperTest {
         Optional<Book> book = bookMapper.findByBookId(0);
         assertThat(book).isEmpty();
     }
+
+    @Test
+    @DataSet("datasets/books.yml")
+    @Transactional
+    void 存在するカテゴリーのIDを指定したときに正常にカテゴリーのデータを返す () {
+        Optional<Category> category = bookMapper.findByCategoryId(1);
```

Comment:
実装面の話ですが、CategoryMapperを別途作ってもいいのかもなとは思いました。
ただ、複数テーブルがあるときにどうクラス設計するかまでは今回のカリキュラムでは考えなくてもいいかなとおもったのでこのままでもよいです。
色々終わって余裕があれば考えればいいですし、何なら現場に入っていろいろな実装方法を学んだ後でもいいと思います。
```suggestion
        Optional<Category> category = categoryMapper.findByCategoryId(1);
```